app/components/ui/create-tale-modal/component.js

```diff
@@ -156,9 +155,8 @@ export default Component.extend({
       self.closeModal();
 
       if (self.createAndLaunch) {
-        let newInstance = store.createRecord('instance');
-        return newInstance.save({adapterOptions:{queryParams:{imageId, taleId}}}).then(instance => {
-          this.newInstance = instance;
+        this.get('apiCall').startTale(this.newTale).then((instance) => {
+          newTale.set('instance', instance)
```

Comment:
I think that manually calling `waitForInstance` is necessary after calling `startTale`, otherwise the instance polling will not start.

Reply:
Is it okay if we defer that to another PR? We'll also want to do this when we import Tales in the same component. Unless I've broken it in this PR, then I'm more than happy to do it here

Reply:
The more I look at this, the more I can't help but think that this issue was purely a side-effect of the missing poller (#531). I am confused on how displaying the Stop button is the correct fix for this issue if we know the button won't work.

I think I now understand how I may have broken this, and I suspect that this what happened:
1. In PR #517, the poller was replaced with a call to `router.transitionTo` with the assumption that navigating to a new view would trigger a refresh and/or activate the poller, so the explicit calls to `waitForInstance` were removed - this was a obviously mistake
2. After navigation, the poller is no longer automatically refreshing instance state behind the scenes. Because we are no longer polling for the instance state (which controls the state of the "Run" button), the user's view never updates to show the started instance unless they navigate themselves (triggering a refresh)
3. Finally, we still get the event in the NotificationStream telling us that the container has started, but the component never fetches the new instance status and as a result the view never updates

As far as I can tell, activating the poller in all applicable cases will fix this issue. I am also a bit wary of removing the `fetchTaleData` stuff right now, as it is unclear if it is in fact still needed and/or if removing it would have any negative side-effects.

Reply:
I've started testing this on another branch here: https://github.com/whole-tale/dashboard/compare/master...fix-instance-poller

I may have still missed triggering the poller in one or more cases, such as import, but will try to gather them all there. There are a lot of these cases, and they're all unfortunately (and usually unnecessarily) unique.